pages/shared/studentAssessmentInstance.js

```diff
@@ -0,0 +1,28 @@
+const _ = require('lodash');
+
+const fileStore = require('../../lib/file-store');
+
+module.exports = {
+    async processFileUpload(req, res) {
+        if (!res.locals.assessment_instance.open) throw new Error(`Assessment is not open`);
+        await fileStore.upload(req.file.originalname, req.file.buffer, 'student_upload', res.locals.assessment_instance.id, null, res.locals.user.user_id, res.locals.authn_user.user_id);
```

Comment:
I'm going to assume that our upload middleware will handle capping the size of uploaded files?

Reply:
yep, it already does (`multer` handles this in `server.js`)